jobs/transforms/audits_registry.py

```diff
@@ -41,3 +42,25 @@ def get_audit_do_fn_types_by_kind():
         targeted.
     """
     return audit_decorators.AuditsExisting.get_audit_do_fn_types_by_kind()
+
+
+def get_id_property_targets_by_kind():
```

Comment:
I thought of another name -- get_all_referenced_models_by_kind(), perhaps? (i.e. "referenced models")

Or possibly get_all_referenced_model_types_by_kind() or get_all_referenced_model_kinds_by_kind() -- I am not sure exactly what the inner tuple(str) represents. Kinds? If so, I'd probably go for get_all_referenced_model_kinds_by_kind.

Note that it's get_all not get -- the latter would typically expect a "kind" argument if named this way.

Reply:
Hmm, but the property that is referencing the model is the most important part of the return value, so I don't think we should remove it from the name. I'm also still not convinced that ID property is a bad name :(

Reply:
To be clear -- I don't think ID property is a bad name either, and it can be used as shorthand for the concept of "IDs of referenced models".

Ah, I didn't realize that the properties were that important in the return value, since the function name focuses more on the "targets". Perhaps something like get_all_id_properties_and_referenced_kinds_by_kind()?

Reply:
OK, how are these names?